syntax = "proto3";


package torch2trt;

enum DataTypeMessage {
  kFloat = 0;
  kHalf = 1;
  kInt8 = 2;
  kInt32 = 3;
}


message interpolate_Message {
  repeated int64 size = 1;
  string mode = 2;
  bool align_corners = 3;
  
  // below params are configured by TRT and not set by user
  DataTypeMessage dtype = 4;
  repeated int64 input_size = 5;
  repeated int64 output_size = 6;
}
